internal/testutil/baseline/baseline.go

```diff
@@ -191,24 +189,16 @@ func writeComparison(t *testing.T, actualContent string, local, reference string
 			}
 		}
 
-		relReference, err := filepath.Rel(repo.RootPath, reference)
-		assert.NilError(t, err)
-		relReference = tspath.NormalizeSlashes(relReference)
-
-		relLocal, err := filepath.Rel(repo.RootPath, local)
-		assert.NilError(t, err)
-		relLocal = tspath.NormalizeSlashes(relLocal)
-
 		if _, err := os.Stat(reference); err != nil {
 			if comparingAgainstSubmodule {
-				t.Errorf("the baseline file %s does not exist in the TypeScript submodule", relReference)
+				t.Errorf("the baseline file %s does not exist in the TypeScript submodule", reference)
```

Comment:
I changed this so the path would always be linkified in VS Code output windows.

Reply:
It was already linkified for me...

Reply:
Not on Windows 🥲